Resources/Locale/en-US/botany/components/plant-analyzer-component.ftl

```diff
@@ -0,0 +1,105 @@
+plant-analyzer-component-no-seed = no plant found
+
+plant-analyzer-component-health = Health:
+plant-analyzer-component-age = Age:
+plant-analyzer-component-water = Water:
+plant-analyzer-component-nutrition = Nutrition:
+plant-analyzer-component-toxins = Toxins:
+plant-analyzer-component-pests = Pests:
+plant-analyzer-component-weeds = Weeds:
+
+plant-analyzer-component-alive = [color=green]ALIVE[color]
+plant-analyzer-component-dead = [color=red]DEAD[color]
+plant-analyzer-component-unviable = [color=red]UNVIABLE[color]
+plant-analyzer-component-mutating = [color=#00ff5f]MUTATING[color]
+plant-analyzer-component-kudzu = [color=red]KUDZU[color]
+
+plant-analyzer-soil = There is some [color=white]{$chemicals}[/color] in this {$holder} that {$count ->
```

Comment:
`There is some` should also be conjugated based on `{$count}`

Reply:
It's always "There is some" for n > 0. For n == 0 it is not displayed at all.